app/controllers/application_controller.rb

```diff
@@ -1,6 +1,16 @@
 class ApplicationController < ActionController::Base
-  helper_method :current_user
-  def current_user
-    @current_user ||= User.first
+  # helper_method :current_user
+  # def current_user
+  #   @current_user ||= User.first
+  # end
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean and tidy :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)